apps/vpnmanager/src/lib/data/spreadsheet.ts

```diff
@@ -45,15 +48,56 @@ async function list(
 }
 
 async function newHires() {
-  const spreadsheetId = process.env.NEXT_APP_GOOGLE_SHEET_ID;
-  const range = process.env.NEXT_APP_GOOGLE_SHEET_RANGE;
   const data = await list(spreadsheetId, range);
   return data.filter(
-    (row: SheetRow) => row.emailAddress && row.keySent !== "Yes",
+    (row: SheetRow) => row.emailAddress && row.keySent?.trim() !== "Yes",
   );
 }
 
+export async function updateSheet(row: Partial<SheetRow>) {
```

Comment:
Does this mean we're updating one row at a time? If yes, isn't there a way of reading a range of rows, update them locally and then write the updated range back to gSheet?

Updating one row at a time feels inefficient and unnecessary (unless it's the only way).